packages/eslint-plugin-pf-codemods/lib/rules/v5/tabs-rename-props.js

```diff
@@ -0,0 +1,14 @@
+const { renameProps } = require("../../helpers");
+
+const renames = {
+  Tabs: {
+    hasSecondaryBorderBottom: "",
+    hasBorderBottom: "hasNoBorderBottom",
```

Comment:
Should this codemod just rename the hasBorderBottom prop? @gitdallas would this be similar to your comment on the isSVG PR? Just wondering if simply renaming it might cause issues i.e. if someone has passed in `hasBorderBottom={false}` to remove the borders, would this then update things so that it becomes `hasNoBorderBottom={false}` and have borders rendered.

Reply:
good point, the boolean value should be flipped

Reply:
good catch/point, the boolean value should be flipped. I think we can still do a code fix like below, wdyt? @thatblindgeye 

`hasBorderBottom` => ""
`hasBorderBottom={true}` => ""
`hasBorderBottom={false}` => `hasNoBorderBottom`
`hasBorderBottom={someVar}` => `hasNoBorderBottom{!someVar}`

Reply:
That sounds good to me! I imagine that'd cover the basic use cases of it.

Reply:
started a slack convo regarding these more complex cases (here)[https://patternfly.slack.com/archives/C048DB6UR2P/p1673450939845399]... but for example, the `someVar` could also be more complex so perhaps may need to be wrapped in `()` in case it is something like `someVar || someOtherVar` ... also trying to get a feel for if we should even be taking on these complex situations vs alerting the consumer and asking them to do it themselves.